app/web/readme.md

```diff
@@ -43,6 +44,8 @@ npm install --global yarn
 
 ## Download & extract the latest protos
 wget -qO- https://develop--protos.preview.couchershq.org/ts.tar.gz | tar xz
+## Or run the following (if you don't have wget or don't want to install it)
+curl -SL https://develop--protos.preview.couchershq.org/ts.tar.gz | tar xz
```

Comment:
Do you want to just switch out the `wget` to `curl` to keep it simple? I'd be happy with that, I just thought wget is more common!

Otherwise please comment this line: I'd like this code snippet to just be runnable as-is so experienced devs can just look at the non-commented lines and see what's going on.

Reply:
I think given how common `curl` is, it should be safe to swap out, I'll do that!